package.json

```diff
@@ -7,12 +7,12 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "bin": {
-    "esy": ".bin/esy",
-    "esy.js": ".bin/esy.js",
-    "esyEnv.js": ".bin/esyEnv.js"
+    "esy": ".bin/esy"
   },
   "dependencies": {
+    "flow-remove-types": "^1.0.5",
```

Comment:
This is runtime dep for now, need to precompile sources to strip flow before publishing. At least for now it is easier to experiment with `esy` that way.

The best would be to rewrite `esy` in OCaml/Reason.